.circleci/config.yml

```diff
@@ -263,7 +252,7 @@ jobs:
           root: build
           paths:
             - "*.zip"
-            - "tendermint_linux_amd64"
+            - "lazyledger_linux_amd64"
```

Comment:
```suggestion
            - "lazyledger-core_linux_amd64"
```

Reply:
Thanks. Updated this (and where they are actually generated) in https://github.com/LazyLedger/lazyledger-core/pull/18/commits/3742114bf5977062de17088b631c2ffaeb8292bb